### DISCUSSION TITLE
green it

### DISCUSSION THREAD

**Discussion by briner** [AdnanHodzic/auto-cpufreq#589] [General]
Hi,

Do you think that we could use such technology to save energy in a datacenter by installing it on virtual machines. Because, as I understand this nice piece of software is only dedicated at laptops. Is their any chance that this could work for simple web servers ?

**Comment by AdnanHodzic** [comment]
Absolutely, and I'm literally putting this as an idea as part of auto-cpufreq v2.0 video: https://youtu.be/SPGpkZ0AZVU?si=tBgJdLBbvtaCltY3&t=152
